osu.Game.Rulesets.Osu/UI/OsuResumeOverlay.cs

```diff
@@ -82,7 +96,7 @@ public bool OnPressed(OsuAction action)
                     case OsuAction.RightButton:
                         if (!IsHovered) return false;
 
-                        this.ScaleTo(new Vector2(2), TRANSITION_TIME, Easing.OutQuint);
+                        this.ScaleTo(2 * CursorScale, TRANSITION_TIME, Easing.OutQuint);
```

Comment:
this should not be a thing. make a container to apply the local scale to and set the other component externally.

Reply:
Changed it, please check if this is how you wanted it.